.idea/codeStyles/Project.xml

```diff
@@ -1,7 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <option name="RIGHT_MARGIN" value="100" />
-    <option name="ENABLE_SECOND_REFORMAT" value="true" />
```

Comment:
Was this intentional?

Reply:
No, probably my upgrade of IntelliJ, I will revert this.